products/endpoints/frontend/EndpointConfiguration.tsx

```diff
@@ -25,14 +26,35 @@ const CACHE_AGE_OPTIONS: { value: CacheAgeOption; label: string }[] = [
     { value: 259200, label: '3 days' },
 ]
 
+const SYNC_FREQUENCY_OPTIONS: { value: DataWarehouseSyncInterval; label: string }[] = [
```

Comment:
i'm a little surprised this doesn't already exist somewhere. just encourage to double check before merging but non-blocking

Reply:
if you're just dropping fields use Pick<> if not no worries!